diff -Nur libstrfunc-7.1.0/Makefile.in libstrfunc-7.1.0.new/Makefile.in
--- libstrfunc-7.1.0/Makefile.in	Tue May 15 17:20:31 2001
+++ libstrfunc-7.1.0.new/Makefile.in	Sun May 20 18:02:12 2001
@@ -73,11 +73,12 @@
 	@echo "Istalling the library and header files..."
 
 install-lib: echo-ilib
-	$(LIBTOOL) ${INSTALL} -m 444 ./.libs/lib${LIB}.a ${libdir}
-	rm -f ${libdir}/${SHLIB_NAME} ${libdir}/${SHLIB_LINK}
-	$(LIBTOOL) ${INSTALL} -m 444 ./.libs/${SHLIB_NAME} ${libdir}
-	${LN_SF} ${SHLIB_NAME} ${libdir}/${SHLIB_LINK}
-	${INSTALL} -m 444 strfunc.h ${includedir}
+	install -d $(DESTDIR)${libdir}
+	$(LIBTOOL) ${INSTALL} -m 444 ./.libs/lib${LIB}.a $(DESTDIR)${libdir}
+	rm -f $(DESTDIR)${libdir}/${SHLIB_NAME} $(DESTDIR)${libdir}/${SHLIB_LINK}
+	$(LIBTOOL) ${INSTALL} -m 444 ./.libs/${SHLIB_NAME} $(DESTDIR)${libdir}
+	${LN_SF} ${SHLIB_NAME} $(DESTDIR)${libdir}/${SHLIB_LINK}
+	${INSTALL} -m 444 strfunc.h $(DESTDIR)${includedir}
 
 install-man:
 	cd man; make install-man
diff -Nur libstrfunc-7.1.0/bin/Makefile.in libstrfunc-7.1.0.new/bin/Makefile.in
--- libstrfunc-7.1.0/bin/Makefile.in	Tue May 15 17:20:31 2001
+++ libstrfunc-7.1.0.new/bin/Makefile.in	Sun May 20 18:03:01 2001
@@ -26,7 +26,8 @@
 	${CC} ${CFLAGS} -o posdump posdump.c ${LIBS}
 
 install: all
-	${INSTALL} -m 555 ${TARGETS} ${bindir}
+	install -d $(DESTDIR)${bindir}
+	${INSTALL} -m 555 ${TARGETS} $(DESTDIR)${bindir}
 
 clean:
 	rm -f *.o *.core 1
diff -Nur libstrfunc-7.1.0/man/Makefile.in libstrfunc-7.1.0.new/man/Makefile.in
--- libstrfunc-7.1.0/man/Makefile.in	Tue May 15 17:20:31 2001
+++ libstrfunc-7.1.0.new/man/Makefile.in	Sun May 20 18:01:28 2001
@@ -14,7 +14,7 @@
 NROFF= groff -Tascii -mandoc
 INSTALL=@INSTALL@
 MANROOT=@mandir@/cat
-MAN3= ${MANROOT}3
+MAN3= $(DESTDIR)${MANROOT}3
 LN= ln
 LN_F= ${LN} -f
 LN_S= @LN_S@
